src/main/resources/assets/idwtialsimmoedm/lang/ru_ru.json

```
@@ -0,0 +1,47 @@
+  "enchantment.minecraft.fire_protection.desc": "Снижает урон от огня и время горения игрока.",
+  "enchantment.minecraft.feather_falling.desc": "Снижает урон от падения и телепортации с помощью эндер-жемчуга.",
+  "enchantment.minecraft.blast_protection.desc": "Снижает урон от взрывов и эффект откидывания при взрыве.",
+  "enchantment.minecraft.projectile_protection.desc": "Снижает урон от таких снарядов, как стрелы и огненные шары.",
+  "enchantment.minecraft.respiration.desc": "Улучшает задержку дыхания и видимость под водой.",
+  "enchantment.minecraft.aqua_affinity.desc": "Увеличивает скорость копания под водой.",
+  "enchantment.minecraft.thorns.desc": "С некоторым шансом наносит урон атакующим.",
+  "enchantment.minecraft.sharpness.desc": "Увеличивает урон.",
+  "enchantment.minecraft.smite.desc": "Увеличивает урон против нежити, таких как зомби и скелеты.",
+  "enchantment.minecraft.bane_of_arthropods.desc": "Увеличивает урон против членистоногих, таких как пауки и чешуйницы.",
+  "enchantment.minecraft.knockback.desc": "Увеличивает силу отбрасывания.",
+  "enchantment.minecraft.fire_aspect.desc": "Поджигает цель.",
+  "enchantment.minecraft.looting.desc": "Увеличивает выпадение добычи с убитых существ.",
+  "enchantment.minecraft.efficiency.desc": "Увеличивает скорость копания инструмента.",
+  "enchantment.minecraft.silk_touch.desc": "Позволяет добывать хрупкие блоки, такие как стекло и не только.",
+  "enchantment.minecraft.unbreaking.desc": "Позволяет инструменту терять прочность с меньшей скоростью.",
+  "enchantment.minecraft.fortune.desc": "Увеличивает шанс выпадения большего количества ресурсов с блоков руд.",
+  "enchantment.minecraft.power.desc": "Увеличивает урон стрел, выпускаемых из лука.",
+  "enchantment.minecraft.punch.desc": "Увеличивает расстояние, на которое будет отброшен противник при попадании в него стрелы.",
+  "enchantment.minecraft.flame.desc": "Выпущенные из лука стрелы будут наносить наносить дополнительный урон огнем.",
+  "enchantment.minecraft.infinity.desc": "Позволяет луку стрелять бесконечно, не тратя стрелы. Но для стрельбы необходима хотя бы одна стрела.",
+  "enchantment.minecraft.luck_of_the_sea.desc": "Увеличивает шанс получения хорошей добычи во время рыбалки.",
+  "enchantment.minecraft.lure.desc": "Снижает количество времени, которое требуется рыбе, чтобы заглотить крючок.",
+  "enchantment.minecraft.depth_strider.desc": "Увеличивает скорость передвижения под водой.",
+  "enchantment.minecraft.frost_walker.desc": "Замораживает воду под игроком.",
+  "enchantment.minecraft.mending.desc": "Использует опыт для починки какого-либо предмета в руках или в слотах брони.",
+  "enchantment.minecraft.binding_curse.desc": "Данное зачарование на предмете не позволит вам выбросить или снять его со слота брони, выпадает из игрока только в случае его смерти.",
+  "enchantment.minecraft.vanishing_curse.desc": "Предмет с данным зачарованием самоуничтожится при смерти игрока.",
```

Улучшенная формулировка
```suggestion
  "enchantment.minecraft.vanishing_curse.desc": "Исчезновение предмета после смерти игрока.",
```